crates/flowctl/src/raw/capture.rs

```diff
@@ -0,0 +1,254 @@
+use std::sync::{Arc, Mutex};
+
+use futures::channel::mpsc::{Sender};
+use futures::{StreamExt, channel, stream};
+use futures::executor::ThreadPool;
```

Comment:
this is running in the context of a tokio runtime -- use that, rather than `futures`, for spawning tasks.